### DISCUSSION TITLE
Can't access github enterprise repositories(403 whitelist.)

### DISCUSSION THREAD

**Discussion by seongwoo90** [community/community#102875] [Enterprise]
that was working well suddenly can't access our repositories.

we used whitelist but such as can't access as below error

![image](https://github.com/community/community/assets/117069979/797c24a3-d898-46a8-bb50-7b22d89a65bb)

**Comment by helixzz** [comment]
Seems it's an issue on GitHub and it's been solved for now.
https://www.githubstatus.com/incidents/5y8b8lsqbbyq
